<ui version="4.0" >
 <class>UIXUPFindFiles</class>
 <widget class="QDialog" name="UIXUPFindFiles" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Choose a file to open</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="lTitleTop" >
     <property name="text" >
      <string>Here is the search result for finding file :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lFindFile" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>146</red>
           <green>145</green>
           <blue>144</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lwFiles" />
   </item>
   <item>
    <widget class="QLabel" name="lTitleBottom" >
     <property name="text" >
      <string>Absolute File Path :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lAbsoluteFilePath" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>146</red>
           <green>145</green>
           <blue>144</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UIXUPFindFiles</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>239</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel" >
     <x>245</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UIXUPFindFiles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>172</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel" >
     <x>183</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
